docs/modules/ROOT/pages/reference/extensions/core.adoc

```diff
@@ -239,6 +239,14 @@ What to do if it is not possible to extract CSimple expressions from a route def
 | `org.apache.camel.quarkus.core.CamelConfig.FailureRemedy`
 | `warn`
 
+|icon:lock[title=Fixed at build time] [[quarkus.camel.event-bridge.enabled]]`link:#quarkus.camel.event-bridge.enabled[quarkus.camel.event-bridge.enabled]`
+
+Whether to enable the bridging of Camel events to CDI events. 
+ This allows CDI observers to be configured for Camel events. E.g. those belonging to the `org.apache.camel.quarkus.core.events`, `org.apache.camel.quarkus.main.events` & `org.apache.camel.impl.event` packages. 
+ Note that his configuration item only has any effect when observers configured for Camel management events are present in the application.
+| `boolean`
+| `true`
```

Comment:
I would say it's ok to keep true as default. My understanding is that we pay a performance price only when users explicitly configure observers.

Reply:
Correct. Arguably you could not have a config param at all and just determine it based on whether observers are present. But I thought it'd still be worth having some option of forcably disabling the event bridge if required.